Tests/Rules/TrailingCommasTests.swift

```diff
@@ -272,4 +272,60 @@ class TrailingCommasTests: XCTestCase {
         let options = FormatOptions(trailingCommas: false)
         testFormatting(for: input, output, rule: .trailingCommas, options: options)
     }
+
+    func testTrailingCommasAddedToFunctionParameters() {
+        let input = "func foo(\n    bar _: Int\n) {}"
+        let output = "func foo(\n    bar _: Int,\n) {}"
```

Comment:
Could you update the test cases to use multiline strings?

```suggestion
        let input = """"
        func foo(
            bar _: Int
        ) {}
        """
        let output = """
        func foo(
            bar _: Int,
        ) {}
        """
```

Reply:
Sure! I did